src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -795,18 +787,9 @@ public Version ProtocolVersion
             }
         }
 
-
-        public int ReadWriteTimeout
-        {
-            get
-            {
-                throw NotImplemented.ByDesignWithMessage(SR.net_PropertyNotImplementedException);
-            }
-            set
-            {
-                throw NotImplemented.ByDesignWithMessage(SR.net_PropertyNotImplementedException);
-            }
-        }
+        // .Net Framework behavior difference: ReadWriteTimeout cannot be changed.
+        // For app-compat reasons we allow apps to change it and return the .Net Framework default.
+        public int ReadWriteTimeout { get; set; } = 300000;
```

Comment:
Nit: desktop has a named const for the value; consider doing that here:
https://referencesource.microsoft.com/#System/net/System/Net/HttpWebRequest.cs,115